Resources/public/Admin.js

```diff
@@ -103,6 +103,31 @@ var Admin = {
                 checkboxClass: 'icheckbox_square-blue',
                 radioClass: 'iradio_square-blue'
             });
+
+            var previousIndex;
+
+            jQuery('tbody .iCheck-helper', subject).click(function (evt) {
```

Comment:
Ok, I think I will extract it, so we could handle checkboxes when iCheck is disabled too !